uint8 error_interval
---
